src/unity/python/turicreate/data_structures/sarray.py

```diff
@@ -4399,6 +4399,92 @@ def cumulative_var(self):
         agg_op = "__builtin__cum_var__"
         return SArray(_proxy = self.__proxy__.builtin_cumulative_aggregate(agg_op))
 
+    def filter_by(self, values, exclude=False):
+        
+        """
+        Filter an SArray by values inside an iterable object. Result is an SArray that
+        only includes (or excludes) the values in the given ``values`` :class:`~turicreate.SArray`.
```

Comment:
The doc string I used here was the same as the one in the original filter_by() in the SFrame. I switched out the SFrame column for SArray wherever applicable since we're using the same logic/code to create this function. That's why I missed removing ":class:~turicreate.SArray", as you mentioned it makes sense to not have it here. Ty for the other feedback too, I'll make those edits as well.